financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsManifestRepository.kt

```diff
@@ -60,6 +61,29 @@ internal interface FinancialConnectionsManifestRepository {
         clientSecret: String
     ): FinancialConnectionsSessionManifest
 
+    @Throws(
+        AuthenticationException::class,
+        InvalidRequestException::class,
+        APIConnectionException::class,
+        APIException::class
+    )
+    suspend fun postAuthorizationSession(
+        clientSecret: String,
+        institutionId: String
+    ): FinancialConnectionsAuthorizationSession
+
+    @Throws(
+        AuthenticationException::class,
+        InvalidRequestException::class,
+        APIConnectionException::class,
+        APIException::class
+    )
+    suspend fun completeAuthorizationSession(
+        clientSecret: String,
+        sessionId: String,
+        publicToken: String? = null
+    ): FinancialConnectionsAuthorizationSession
```

Comment:
These are all api calls that receive an updated auth session. Moved them to the `manifestRepository` so that we can update the cached version internally.